elasticassignment/ageofempires/management/commands/index_civilizations.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+import ageofempires.models.civilizations as Civilizations
+from django.conf import settings
+
+class Command(BaseCommand):
```

Comment:
- create single command `import_data` for all imports and this command should accept `import_name` param.
- keep the `import_name` param optional. If not given, all imports should execute.

Reply:
- Create a command `create_indexes` which should create all indexes for the app.

Reply:
- In `Elastic_search` package, please add a module `index_config`.
- add `get_index_settings` method.
- add `get_index_mappings` method which should accept `index_name` param.